wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -91,7 +92,7 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
 **Your construction toolkit, made for DIY (do it yourself).**
 
-**Please note that in osu!standard's default notes hitsounds works differently in osu!mania.**
+**Please note that in the osu!'s mode default notes hitsounds works differently in osu!mania.**
```

Comment:
This isn't good as an english sentence. The original *and* this change are wrong.